WG_JS_CORE.md

```diff
@@ -97,34 +96,92 @@ Improving the docs, specs and having great tutorials and examples goes a long wa
   - Return values are documented, including object properties and types
 - Onboarding documentation exists to explain domain concepts and provide tutorials/examples for achieving common tasks
 
-### `M (P0)`: Revamped APIs 🔄 🤝 🧠
+### `M (P0)`: Revamped APIs 📦 🤝
 
 Revamping the APIs will make them easier to use and simpler to understand. Using modern syntax will make JS IPFS appealing to a wider segment of the JS community, aiding adoption and contributions. It also gives us scope to make performance optimisations and stability/debuggability improvements.
 
 - Async/await is used throughout the codebase and callback APIs are dropped
-- Support ArrayBuffer as an alternative to Buffer input argument
+- ArrayBuffer is supported as an alternative to Buffer input argument
 - Streaming APIs switch to using async iterators/iterables
 - The files API is revamped:
     - Files functions moved to the root namespace
     - Non-MFS and MFS functions are merged
     - A revamped RESTful HTTP API exists
 - Old APIs are deprecated and removed (e.g. object, block)
-- At least one alternative API exists for JS IPFS (e.g. Graphql)
-
-### `M (P3)`: Support for specific use cases 🔄 🤝 🧠
-
-This milestone focuses in on how JS IPFS can refine and augment it’s existing tech stack to better serve specific use cases.
-
-- Default chunkers exist for particular file types to optimise for example deduplication or seeking/direct access in tar archives and video files
 - High priority features requested from other working groups that have experienced pain points with the JS IPFS API are implemented e.g. Pagination and “Live” streams
-- IPFS nodes can discover one another for specific topics via the rendezvous protocol
--star servers are no longer in use
-- IPFS nodes work and are compliant with the MDNS spec
-- Libp2p discovery and transport modules exist for bluetooth and libdweb, sound and potentially other exotic technologies
 
 ## Timeline
```

Comment:
@achingbrain @daviddias I've had a go at organising the roadmap milestones and goals into a timeline (below). Please take a look and let me know if it looks sensible - I've tried to leave Q4 with slightly fewer tasks than the preceeding quarters so we have a bit of a buffer for overflow. Q1 is essentially stuff I know is in progress or done.

cc-ing @vasco-santos because I believe gossipsub is on your libp2p roadmap - does that align with when you were hoping to look at it?

Reply:
It looks good but there's quite a lot in Q2.